scripts/recover-sqlite-from-backups.sh

```
@@ -0,0 +1,239 @@
+  echo "  ${REMOTE_DB_DIR}/${fname}"
+  echo "Перед заменой будет создан дополнительный .bak рядом."
+  if [ "${BIRDLENSE_RESTORE_CONFIRM_YES:-}" = "1" ]; then
+    confirm="YES"
+  else
+    read -r -p 'Введите YES для продолжения: ' confirm
+  fi
+  if [ "${confirm}" != "YES" ]; then
+    echo "Отмена."
+    exit 1
+  fi
+  ssh ${SSH_OPTS} "${HOST}" bash -s -- "${REMOTE_DB_DIR}" "${fname}" <<'REMOTE'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)